### DISCUSSION TITLE
Log entry length limit?

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#21230] [Q&A] [answered]
I am getting an error when trying to export a skin, but the error message in the log is cut off with three dots at the end, is there a length limit on individual log entries?

**Comment by smoogipoo** [comment, marked as answer]
For messages logged in-game, yes. But the full message should be present in the runtime.log file.

**Reply by SevenChords** [reply]
that's the thing, even in the runtime.log file the message is cut off with three dots

**Reply by frenzibyte** [reply]
Please attach the logs file to further determine what went wrong.

**Reply by SevenChords** [reply]
nvm, the actual error message is not cut off, it's just the `[verbose]` message that got cut off, i'm assuming that's the one shown in-game then
![image](https://user-images.githubusercontent.com/31819497/202035591-c00dd355-b9e8-4e09-b76b-58474a357946.png)

**Reply by frenzibyte** [reply]
The one with the warning icon happens as a result of an important notification being posted, that one will get trimmed.
